### PR TITLE
Tweak pnpm guide

### PR DESCRIPTION
**Pull request by samchungy**
Adjust guide based on feedback from the team.

### CODE REVIEW

File: docs/deep-dives/pnpm.md [hunk 1]
```diff
@@ -289,10 +289,10 @@ This migration guide assumes that your project was scaffolded with a **skuba** t
     - seek-oss/docker-ecr-cache#v2.1.0:
     + seek-oss/docker-ecr-cache#v2.2.0:
         cache-on:
-    +     - .npmrc
     -     - package.json
-    +     - package.json#.packageManager
     -     - yarn.lock
+    +     - .npmrc
+    +     - package.json#.packageManager
```
Comment (samchungy) [inline docs/deep-dives/pnpm.md:295]: Optimise for copy paste replacement

File: docs/deep-dives/pnpm.md [hunk 2]
```diff
@@ -263,7 +263,7 @@ This migration guide assumes that your project was scaffolded with a **skuba** t
       WORKDIR /workdir
     
       COPY --from=build /workdir/lib lib
-    -
+    - COPY --from=build /workdir/packages packages
```
Comment (samchungy) [inline docs/deep-dives/pnpm.md:266]: This shouldn't affect the majority but just adds an extra monorepo use case similar to the dev deps dockerfile changelog.

**Review by AaronMoat — APPROVED** [review]
Should we recommend mise while we're here?

File: docs/deep-dives/pnpm.md [hunk 3]
```diff
@@ -316,21 +316,25 @@ This migration guide assumes that your project was scaffolded with a **skuba** t
...
 
-    ***
+**A** Ensure `pnpm install --offline` is referenced earlier within pipeline step as shown in step 14.
 
-15. Search for other references to `yarn` in your project. Replace these with `pnpm` where necessary.
+---
+
+**Q** I'm seeing `ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command "workspace" not found` in my pipeline
+
+**A** `pnpm workspace <PACKAGE_NAME>` does not work. Replace it with the [`filter`](https://pnpm.io/filtering) flag.
```
Comment (72636c) [inline docs/deep-dives/pnpm.md:337]:
```suggestion
**A** `pnpm workspace <PACKAGE_NAME>` does not work. Replace it with the [`--filter`](https://pnpm.io/filtering) flag.
```

File: docs/deep-dives/pnpm.md [hunk 4]
```diff
@@ -316,21 +316,25 @@ This migration guide assumes that your project was scaffolded with a **skuba** t
...
+15. Search for other references to `yarn` in your project. Replace these with `pnpm` where necessary.
 
-    **Q:** I'm running into `ERR_PNPM_CANNOT_DEPLOY  A deploy is only possible from inside a workspace`
+## FAQ
 
-    **A:** `pnpm deploy` is a reserved command. Use `pnpm run deploy` instead.
+**Q:** I'm running into `ERR_PNPM_CANNOT_DEPLOY  A deploy is only possible from inside a workspace`
 
-    ***
+**A:** `pnpm deploy` is a reserved command. Use `pnpm run deploy` instead.
```
Comment (72636c) [inline docs/deep-dives/pnpm.md:325]: Nitpick: do we want the colons

```suggestion
**Q** I'm running into `ERR_PNPM_CANNOT_DEPLOY  A deploy is only possible from inside a workspace`

**A** `pnpm deploy` is a reserved command. Use `pnpm run deploy` instead.
```

**Review by 72636c — APPROVED** [review]

**Pull request merged**